From: Michael Froman <mjfroman@mac.com>
Date: Tue, 18 Jun 2024 17:34:08 -0500
Subject: (tmp-cherry-pick) Revert "lets try again" (cfddbfeae0)

This reverts commit f03b06e3ac225a2b4fabd7c9e85bf844d065581f.

Reason for revert: Testing again

Original change's description:
> lets try again
>
> No-Try: True
> Bug: webrtc:337093715, webrtc:337095649, b:337172394
> Change-Id: Id8afe5e0aba6f64314527fe14a3631566cbc34ee
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/349261
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Owners-Override: Mirko Bonadei <mbonadei@webrtc.org>
> Commit-Queue: Christoffer Dewerin <jansson@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#42180}

Bug: webrtc:337093715, webrtc:337095649, b:337172394
Change-Id: If94f5b9d2747d4c77606445dcfc03cc6f67a6cf8
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/349262
Commit-Queue: Christoffer Dewerin <jansson@webrtc.org>
Owners-Override: Christoffer Dewerin <jansson@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: Christoffer Dewerin <jansson@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#42181}
---
 whitespace.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/whitespace.txt b/whitespace.txt
index 35eb19d4bc..389c15fecc 100644
--- a/whitespace.txt
+++ b/whitespace.txt
@@ -6,4 +6,3 @@ Once upon a time there was a white elephant in Stockholm.
 Why did the elephant get kicked out of the Swedish Parliament?
 Because it kept making trunk calls!
 Migration is done, we are now testing!
-Let's try again!
